packages/web-service/src/pages/method-statement/upload-method-statement.js

```diff
@@ -1,11 +1,16 @@
+import { s3FileUpload } from '../../services/s3-upload.js'
 import { FILE_UPLOADS } from '../../uris.js'
-import { fileUploadPageRoute } from '../common/file-upload/file-upload.js'
+import { FILETYPES, fileUploadPageRoute } from '../common/file-upload/file-upload.js'
 
 export const completion = async request => {
   const journeyData = await request.cache().getPageData() || {}
+  const { applicationId, fileUpload } = await request.cache().getData()
   if (journeyData.error) {
     return FILE_UPLOADS.METHOD_STATEMENT.FILE_UPLOAD.uri
```

Comment:
Can you let me know why -- normally errors are handled in the validator

Reply:
I agree. As a result, I have removed the condition of checking error as it has been taken care of by the validator on `fileUploadPageRoute`.